packages/utilities/date-utils/src/get-month-days.ts

```diff
@@ -16,8 +16,18 @@ export function getDaysInWeek(weekIndex: number, from: DateValue, locale: string
   return dates
 }
 
+export function getCustomWeeksInMonth(from: DateValue, locale: string, firstDayOfWeek?: number) {
+  if (firstDayOfWeek == null) {
+    return getWeeksInMonth(from, locale)
+  }
+  const paddingDays = (getDayOfWeek(startOfMonth(from), "en") - firstDayOfWeek + 7) % 7
```

Comment:
I believe this should use the locale instead of the hardcoded 'en'? 

```suggestion
  const paddingDays = (getDayOfWeek(startOfMonth(from), locale) - firstDayOfWeek + 7) % 7
```

Reply:
`getDayOfWeek` returns a number depending on locale. So `0` always matches the locale's week start. https://github.com/adobe/react-spectrum/blob/main/packages/%40internationalized/date/src/queries.ts#L72

The `firstDayOfWeek` and other code expects 0 to always be Sunday, so I chose "en" but it can be arbitrary. I couldn't figure out an simpler alternative since all the internationalized functions are relative to a locale. I can add a comment explaining.